types/sdk.js

```diff
@@ -60,7 +60,7 @@ export type AuthMiddlewareOptions = {
   scopes: Array<string>;
 }
 export type HttpMiddlewareOptions = {
-  host?: string;
+  host: string;
```

Comment:
Playing the "devil's advocate" here: what type definition is 'correct'. If looking it from the other way putting a Maybe type on it is not correct since we set it to a String if nothing is passed.